contributing/translations/index.rst

```diff
@@ -0,0 +1,90 @@
+Contributing Translations
+=========================
+
+Some Symfony Components include certain messages that must be translated to
+different languages. For example, if a user submits a form with a wrong value in
+a :doc:`TimezoneType </reference/forms/types/timezone>` field, Symfony shows by
+default the following error message: "This value is not a valid timezone."
+
+These messages are translated into tens of languages thanks to the Symfony
+community. Symfony adds new messages on a regular basis, so this is an ongoing
+translation process and you can help us providing the missing translations.
+
+How to Contribute a Translation
+-------------------------------
+
+Imagine that you can speak both English and Swedish and want to check if there's
+some missing Swedish translations to contribute them.
+
+**Step 1.** Translations are contributed to the oldest maintained branch of the
+Symfony repository. Visit the `Symfony Releases`_ page to find out which is the
+current oldest maintained branch.
+
+If you know Git well, clone the Symfony repository and check out that branch.
+Otherwise, visit https://github.com/symfony/symfony and switch to that branch.
+
+**Step 2.** Check out if there's some missing translation in your language.
+You only have to check these directories:
+
+* ``src/Symfony/Component/Form/Resources/translations/``
+* ``src/Symfony/Component/Security/Core/Resources/translations/``
+* ``src/Symfony/Component/Validator/Resources/translations/``
+
+Symfony uses the :ref:`XLIFF format <best-practice-internationalization>` to
+store translations. In this example, you are looking for missing Swedish
+translations, so you should look for files called ``*.se.xlf``.
+
+.. note::
+
+    If there's no XLIFF file yet for your language, create it yourself
+    duplicating the original English file (e.g. ``validators.en.xlf``).
+
+**Step 3.** Contribute the missing translations. To do that, compare the file
+in your language to the equivalent file in English.
+
+Imagine that you open the ``validators.sv.xlf`` and see this at the end of the file:
+
+.. code-block:: xml
+
+    <!-- ... -->
+    <trans-unit id="91">
+        <source>This value should be either negative or zero.</source>
+        <target>Detta värde bör vara antingen negativt eller noll.</target>
+    </trans-unit>
+    <trans-unit id="92">
+        <source>This value is not a valid timezone.</source>
+        <target>Detta värde är inte en giltig tidszon.</target>
+    </trans-unit>
+
+If you open the equivalent ``validators.en.xlf`` file, you can see that the
+English file has more messages to translate:
+
+.. code-block:: xml
```

Comment:
```suggestion
.. code-block:: xliff
```

Reply:
I looked for and `xliff` doesn't exist in the Symfony Docs whereas `xml` is common (also, in this example we're just showing a XML file without any XLIFF feature, so this should be fine).